tests/optimizers/test_rdkit.py

```diff
@@ -0,0 +1,42 @@
+import rdkit.Chem as Chem
+from stko.optimizers import MMFF, UFF, ETKDG
```

Comment:
why have you imported these, but used `stko.MMFF()` below? (for example). Make sure your imports are cleaned up.